packages/ollydbg.vm/ollydbg.vm.nuspec

```diff
@@ -2,8 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ollydbg.vm</id>
-    <version>1.10</version>
-    <tags>reverse engineering debugger disassembler binary analysis</tags>
+    <version>1.11</version>
```

Comment:
is this the proper version update?

Reply:
no, you are not updating the tool, so you shouldn't update the first 3 segments as they are the tool version.

From our [documentation](https://github.com/mandiant/VM-Packages/wiki/Contributing#version):

> When do I update the a package's version string?
> 
> The version needs to be increased every time you modify an existing package. For example, if you fix a bug in a package, you need to update the nuspec version string. Our guidance is to add the current date to the 4th segment using the format YYYMMDD. For example, 1.6.2 becomes 1.6.2.20220113 and 3.02 becomes 3.02.0.20220113. If the date is already in the 4th segment, you can increment it to the next day (e.g., 20220113 -> 20220114) and when the PR is merged it should be good to go.

maybe the linter can catch this 🤔

Reply:
61f4a87
I don't think the linter can automatically catch this here.